build.gradle

```diff
@@ -45,6 +45,8 @@ buildscript {
             opensearch_build += "-SNAPSHOT"
         }
         opensearch_no_snapshot = opensearch_build.replace("-SNAPSHOT","")
+        repo_s3_resource_folder = "build/resource/repository-s3"
+        repo_s3_download_url = "https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/3.0.0-alpha1/latest/linux/arm64/tar/builds/opensearch/core-plugins/repository-s3-3.0.0-alpha1.zip"
```

Comment:
Please use relative links to build this URL. This includes versions of OpenSearch and also the distribution URL. Take some reference from here: https://github.com/opensearch-project/k-NN/blob/main/qa/build.gradle#L67-L74